src/js/components/CheckBox/CheckBox.js

```diff
@@ -192,7 +193,7 @@ const CheckBox = forwardRef(
 
     return (
       <StyledCheckBoxContainer
-        aria-label={a11yTitle}
+        aria-label={ariaLabel || a11yTitle}
```

Comment:
I think `aria-label` will work transparently via the `...rest` spread. So I don't think we need an explicit attention for that.

Reply:
StyledCheckBoxContainer doesn't have `{...rest}` on it